invalid memory address or nil pointer dereference upon attempting to write to an unauthorized repo via ssh

See this session:
```
[strk@liz:~/go/src/code.gitea.io/tea(pulls-create)] git push -u origin strk/pulls-create
Gitea: Unauthorized
User: 288:strk with Key: 60:strk@liz is not authorized to write to gitea/tea.
Gitea: Internal Server Error
User: 288:strk with Key: 60:strk@liz is not authorized to write to gitea/tea.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x15ed35f]

goroutine 1 [running]:
code.gitea.io/gitea/cmd.runServ(0xc0008962c0, 0x0, 0x0)
        /go/src/code.gitea.io/gitea/cmd/serv.go:209 +0x90f
github.com/urfave/cli.HandleAction(0x1841660, 0x289d468, 0xc0008962c0, 0xc00018cb00, 0x0)
        /go/src/code.gitea.io/gitea/vendor/github.com/urfave/cli/app.go:490 +0xc8
github.com/urfave/cli.Command.Run(0x1aaca54, 0x4, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1b13c47, 0x2f, 0x0, ...)
        /go/src/code.gitea.io/gitea/vendor/github.com/urfave/cli/command.go:210 +0x996
github.com/urfave/cli.(*App).Run(0xc0001a29c0, 0xc00003c0c0, 0x4, 0x4, 0x0, 0x0)
        /go/src/code.gitea.io/gitea/vendor/github.com/urfave/cli/app.go:255 +0x6af
main.main()
        /go/src/code.gitea.io/gitea/main.go:108 +0x7fa
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

At the time of writing, gitea.com runs Gitea Version: 1.9.0+dev-316-g9002c5157

--- a/cmd/serv.go
+++ b/cmd/serv.go
@@ -53,7 +53,7 @@ func checkLFSVersion() {
 		//Needs at least git v2.1.2
 		binVersion, err := git.BinVersion()
 		if err != nil {
-			fail(fmt.Sprintf("Error retrieving git version: %v", err), fmt.Sprintf("Error retrieving git version: %v", err))
+			fail("LFS server error", "Error retrieving git version: %v", err)
 		}
 
 		if !version.Compare(binVersion, "2.1.2", ">=") {
@@ -199,12 +199,12 @@ func runServ(c *cli.Context) error {
 		if private.IsErrServCommand(err) {
 			errServCommand := err.(private.ErrServCommand)
 			if errServCommand.StatusCode != http.StatusInternalServerError {
-				fail("Unauthorized", errServCommand.Error())
+				fail("Unauthorized", "%s", errServCommand.Error())
 			} else {
-				fail("Internal Server Error", errServCommand.Error())
+				fail("Internal Server Error", "%s", errServCommand.Error())
 			}
 		}
-		fail("Internal Server Error", err.Error())
+		fail("Internal Server Error", "%s", err.Error())
 	}
 	os.Setenv(models.EnvRepoIsWiki, strconv.FormatBool(results.IsWiki))
 	os.Setenv(models.EnvRepoName, results.RepoName)

